groovy/src/main/com/nvuillam/CodeNarcServer.groovy

```diff
@@ -240,6 +243,6 @@ class CodeNarcServer {
         server.stop(1)
         LOGGER.info('Stopped')
         latch.countDown()
-   }
+    }
```

Comment:
question: This looks like an editor format change that isn't needed?

Reply:
Looks like it, will revert to avoid unwanted changes here. ⚙️

Reply:
Fixed, thanks!